src/herder/ParallelTxSetBuilder.cpp

```diff
@@ -351,6 +490,13 @@ buildSurgePricedParallelSorobanPhase(
     std::vector<bool>& hadTxNotFittingLane)
 {
     ZoneScoped;
+    // We prefer the transaction sets that are well utilized, but we also want
+    // to lower the stage count when possible. Thus we will nominate a tx set
+    // that has the lowest amount of stages while still being within
+    // MAX_INCLUSION_FEE_TOLERANCE_FOR_STAGE_COUNT from the maximum total
+    // inclusion fee (a proxy for the transaction set utilization).
+    double const MAX_INCLUSION_FEE_TOLERANCE_FOR_STAGE_COUNT = 0.999;
```

Comment:
How was this value chosen? Seems pretty large.

Reply:
Sorry I wasn't clear - I'm just curious why .1% makes more sense than lets say 1%.

Reply:
It's a bit arbitrary, but the general intention is to just compensate for noise (like if we pick one transaction over another due to a slightly different iteration order). 0.1% just seems pretty non-offensive to me. 1% may be a bit too much (e.g. it would change results of some tests that use relatively large range of fees). This could also be just 1, it probably wouldn't make any difference most of the time. Please let me know if you think that's more appropriate.

Reply:
No it's fine to keep as is.